docs/client/maestro.md

```diff
@@ -4,89 +4,282 @@ title: MAESTRO
+MAESTRO is generally not used in the TMA, however it allows the TCU controllers to have situational awareness of the flow of inbound aircraft.
 
-## Limitations
-VATPAC's implementation of MAESTRO, whilst good enough for VATSIM, has a number of limitations compared to the real world system.
+The MAESTRO plugin can be accessed from the vatSys menu bar under TFMS.
 
-It will not:  
+## Abbreviations
 
-- attempt to organise the sequence in a way to minimize total delay.  
-- provide for dependent or semi-dependent runway modes (when multiple runways are used for arrivals) such as  
-    - Parallel runway operations at Brisbane and Sydney.  
-    - LAHSO at Melbourne (34/27).  
-- provide any wake turbulence separation.
+| Abbreviation | Definition |
+| ------------ | ---------- |
+| Feeder Fix (FF) | A point on the TMA boundary. |
+| `ETA` | Estimated time of arrival **at the runway**. |
+| `STA` | **Scheduled** time of arrival at the runway (landing time) calculated by Maestro. |
+| `ETA_FF` | Estimated time of arrival at the **feeder fix**. |
+| `STA_FF` | **Scheduled** time of arrival at the feeder fix calculated by Maestro. |
 
 ## How it works
 
-### The Basics
-A Feeder Fix (FF) is a common waypoint that aircraft will 'merge' to from a certain direction, commonly (but not always) the first waypoint on a STAR.
+Maestro tracks flights within 2 hours of the feeder fix or when an FDR is activated for flights from departure airports. vatSys provides updated position information every 30 seconds, and Maestro recalculates estimates and sequence position based on the flight's state.
+
+Flights from departure airports are placed in the Pending list and must be manually inserted by the flow controller. These flights can be inserted prior to departure to absorb delay on the ground.
+
+Maestro calculates landing times (`STA`) based on estimates (`ETA`) and applies delays when the time between consecutive flights is less than the acceptance rate. The `STA_FF` is calculated by subtracting the arrival `ETI` from the `STA`.
+
+### The Timeline
+
+Each timeline displays flights using their `STA` (runway view) or `STA_FF` (feeder view). Each tick on the timeline corresponds to one minute.
+
+Flight labels contain (from innermost to outermost):
+
+1. `STA` (in feeder view) or `STA_FF` (in runway view)
+2. Assigned runway
+3. Callsign
+4. `#` if zero delay has been assigned
+5. `%` if manual delay (other than zero) has been assigned
+6. `+` if the flight must cross the feeder-fix at published speed
+7. `*` if the FDR is not coupled to a radar track
+8. Total delay required (based on the initial `ETA`)
+9. Delay remaining (based on the current `ETA`)
+
+The total delay required remains unchanged as the flight absorbs delay. The remaining delay progressively reduces as delay is absorbed. When the remaining delay reads `00`, all required delay has been absorbed.
```

Comment:
Would it be possible to add a screenshot (even just vertically clipped to show a handful of flights) of the ladder in this section?